src/util/paginate-response.ts

```diff
@@ -1,6 +1,7 @@
 import { Workspace } from "../routes/jira/workspaces/jira-workspaces-get";
+import { WorkspaceRepo } from "routes/jira/workspaces/repositories/jira-workspaces-repositories-get";
 
-export const paginatedResponse = (page: number, limit: number, payload: Workspace[]) => {
+export const paginatedResponse = (page: number, limit: number, payload: Workspace[] | WorkspaceRepo[]) => {
```

Comment:
/nit
time to do some generic here :) 
export const paginatedResponse =`<T>` (page: number, limit: number, payload: T[]) => {